modules/openapi-generator/src/main/resources/swift5/Package.swift.mustache

```diff
@@ -35,7 +35,7 @@ let package = Package(
         .target(
             name: "{{projectName}}",
             dependencies: [{{#useAlamofire}}"Alamofire", {{/useAlamofire}}{{#usePromiseKit}}"PromiseKit", {{/usePromiseKit}}{{#useRxSwift}}"RxSwift"{{/useRxSwift}}],
-            path: "{{projectName}}/Classes"
+            path: "{{#useSPMFileStructure}}Sources{{/useSPMFileStructure}}{{^useSPMFileStructure}}{{projectName}}/Classes{{/useSPMFileStructure}}"
```

Comment:
What about adding an option `swiftPackagePath` to customize the whole value instead and default it to `{{projectName}}/Classes` so that users have full control of it?